node/src/blockchain/blockchain_interface.rs

```diff
@@ -1011,6 +1010,10 @@ mod tests {
                 transactions: vec![]
             })
         );
+        let test_log_handler = TestLogHandler::new();
+        test_log_handler.assert_logs_contain_in_order(vec![
```

Comment:
This looks as though it would work fine, but why `.assert_logs_contain_in_order` instead of `.exists_log_containing`?